Plugins/GitUIPluginInterfaces/BuildServerIntegration/IBuildServerAdapter.cs

```diff
@@ -15,5 +15,14 @@ public interface IBuildServerAdapter : IDisposable
         IObservable<BuildInfo> GetFinishedBuildsSince(IScheduler scheduler, DateTime? sinceDate = null);
 
         IObservable<BuildInfo> GetRunningBuilds(IScheduler scheduler);
+
+        /// <summary>
+        /// Called when the current repository is closed
```

Comment:
```suggestion
        ///  Provides an extension point for handling the switch of repositories.
        ///  For example, it could be used to clear build changes.
```